### PR TITLE
EventLogger  class

### PR DESCRIPTION
**Pull request by alszla**

### CODE REVIEW

File: src/main/java/com/foundation/salesforce/core/utils/EventLogger.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    /**
+     * Logs a message at the DEBUG level including the stack trace of the Throwable t passed as parameter.
+     *
+     * @param message A customized message that briefly summarizes the situation.
+     * @param t  An instance of Exception class (or one of its subclasses) thrown by the Java Virtual Machine or
+     *           by the Java throw statement.
+     */
+    public static void debug(String message, Throwable t) {
```
Comment (MelissaRoman06) [inline src/main/java/com/foundation/salesforce/core/utils/EventLogger.java]: Please assign a more descriptive name to t variable.

**Review by titor-oopart — APPROVED** [review]

File: src/main/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#
+# @(#) log4j.properties Copyright (c) 2019 Jala Foundation.
```
Comment (Cxristian) [inline src/main/resources/log4j.properties]: the file with extension properties does not have to have copyright

Comment (alszla) [reply src/main/resources/log4j.properties]: Fixed.

File: src/main/java/com/foundation/salesforce/core/utils/EventLogger.java (hunk 1 above)
Comment (alszla) [reply src/main/java/com/foundation/salesforce/core/utils/EventLogger.java]: Fixed.

**Review by Cxristian — APPROVED** [review]

**Review by MelissaRoman06 — APPROVED** [review]

**Pull request merged**